apps/codeforafrica/src/payload/blocks/Posts.js

```diff
@@ -0,0 +1,70 @@
+const primaryTags = [
+  {
+    label: "Stories",
+    value: "stories",
+  },
+  {
+    label: "Opportunities",
+    value: "opportunities",
+  },
+];
+
+const Posts = {
+  slug: "post-list",
+  fields: [
+    {
+      name: "primaryTag",
+      label: "Primary Tag",
```

Comment:
Label should be something that makes sense to content editors... Post type or something along those lines.